baseapp-reactions/baseapp_reactions/tests/test_notifications.py

```
@@ -0,0 +1,27 @@
+Reaction = swapper.load_model("baseapp_reactions", "Reaction")
+Notification = swapper.load_model("notifications", "Notification")
+
+
+def test_user_receives_notification_when_reaction_is_created(graphql_client):
+    user = UserFactory()
+    target = CommentFactory(user=user)
+
+    with override_settings(BASEAPP_REACTIONS_ENABLE_NOTIFICATIONS=True):
+        with patch(
+            "baseapp_reactions.notifications.send_reaction_created_notification.delay"
+        ) as mock:
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)